net/dpdk/Makefile

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME := dpdk
+PKG_VERSION := 21.05
+
+PKG_SOURCE := dpdk-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL := http://static.dpdk.org/rel
+PKG_HASH := 1e12499b4c5fcdb57c83e5fe184ea6becddf7cf092893c2c5ef2efb0eec12f0b
+
```

Comment:
What about:

```
PKG_CONFIG_DEPENDS:= \
    CONFIG_KERNEL_HUGETLBFS \
    CONFIG_KERNEL_PROC_PAGE_MONITOR \
    CONFIG_KERNEL_HPET \
    CONFIG_KERNEL_HPET_MMAP
```

to require these variables?

https://doc.dpdk.org/guides/linux_gsg/sys_reqs.html